rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -687,7 +548,7 @@ private void highlightGroup(boolean selected) {
 
         @Override
         public boolean equals(Object o) {
-            NodeTile t = (NodeTile) o;
+            Tile t = (Tile) o;
             if (this.node != null && t.node != null) {
                 return this.node.getNodeUrl().equals(t.node.getNodeUrl());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "white" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)